sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/CertificateValidationCheck.CSharp9.cs

```diff
@@ -0,0 +1,149 @@
+﻿
```

Comment:
pedantic: there are a few extra white spaces in this file (e.g. extra empty line on top, trailing whitespace on line 21, 41)

Reply:
thanks, I didn't have the Trailing Whitespace visualizer installed in the preview IDE :)